### PR TITLE
Cisco Umbrella - Added threshold parameter to override the default malicious score 

### PR DESCRIPTION
**Pull request by yardensade**
+ fix to calculate dbotscore properly + test

## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/16244

## Screenshots
image

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Documentation (https://github.com/demisto/etc/issues/16297)
- [x] Code Review

### CODE REVIEW

File: Integrations/integration-Cisco-umbrella.yml [hunk 1]
```diff
@@ -191,9 +196,9 @@ script:
         DBotScore = 0
         if sr > 0 and sr <= 100:
             DBotScore = 1
-        elif sr < 0 and sr > -100:
+        elif sr < 0 and sr > MALICIOUS_THRESHOLD:
             DBotScore = 2
-        elif sr == -100:
+        elif sr < MALICIOUS_THRESHOLD:
```
Comment (omercnet) [inline Integrations/integration-Cisco-umbrella.yml]: <=

File: Integrations/integration-Cisco-umbrella.yml [hunk 2]
```diff
@@ -500,7 +505,7 @@ script:
                 secure_rank = res.get('securerank2', False)
                 DBotScore = 0
                 if secure_rank:
-                    if threshold != DEFAULT_THRESHOLD and secure_rank < threshold:
+                    if threshold != MALICIOUS_THRESHOLD and secure_rank < threshold:
```
Comment (omercnet) [inline Integrations/integration-Cisco-umbrella.yml]: no need for first part of if
if command threshold wins, make it 3

File: Integrations/integration-Cisco-umbrella.yml [hunk 3]
```diff
@@ -1075,7 +1082,7 @@ script:
                 }
                 secure_rank = res.get('securerank2', False)
                 if secure_rank:
-                    if threshold != DEFAULT_THRESHOLD and secure_rank < threshold:
+                    if threshold != MALICIOUS_THRESHOLD and secure_rank < threshold:
```
Comment (omercnet) [inline Integrations/integration-Cisco-umbrella.yml]: same here

File: Integrations/integration-Cisco-umbrella.yml [hunk 4]
```diff
@@ -30,6 +30,11 @@ configuration:
   defaultvalue: https://investigate.api.umbrella.com
   type: 0
   required: true
+- display: DbotScore Malicious Threshold
```
Comment (yaakovi) [inline Integrations/integration-Cisco-umbrella.yml]: should specify a range

Comment (yardensade) [reply Integrations/integration-Cisco-umbrella.yml]:
```suggestion
- display: DbotScore Malicious Threshold (-100 to 100)
```

**Comment by Itay4** [comment]
pending @kirbles19 review

File: Integrations/integration-Cisco-umbrella.yml [hunk 5]
```diff
@@ -30,6 +30,11 @@ configuration:
   defaultvalue: https://investigate.api.umbrella.com
   type: 0
   required: true
+- display: DbotScore Malicious Threshold (-100 to 100)
```
Comment (yardensade) [inline Integrations/integration-Cisco-umbrella.yml]:
```suggestion
- display: DBot Score Malicious Threshold (-100 to 100)
```

**Pull request merged**